### PR TITLE
Add "configure_lustre_network" provision step

### PR DESCRIPTION
**Pull request by johnsonw**
Signed-off-by: Will Johnson <wjohnson@whamcloud.com>

### CODE REVIEW

File: iml-sandbox/Vagrantfile [hunk 1]
```diff
@@ -104,6 +104,18 @@ def fix_hostfile(config)
...
 end
 
+def configure_lustre_network(config)
+  config.vm.provision 'configure-lustre-network', type: 'shell', run: 'never', inline: <<-SHELL
+    systemctl stop firewalld; systemctl disable firewalld;
+    systemctl start ntpd;
+    modprobe lnet
+    lnetctl lnet configure
+    lnetctl net add --net tcp0 --if enp0s9
+    /sbin/modprobe zfs;
```
Comment (jgrund) [inline iml-sandbox/Vagrantfile]: Why are we doing this?

File: iml-sandbox/Vagrantfile [hunk 2]
```diff
@@ -104,6 +104,18 @@ def fix_hostfile(config)
                       inline: "sed  -i '/^127.0.0.1\t#{config.vm.hostname}*/d' /etc/hosts"
 end
 
+def configure_lustre_network(config)
+  config.vm.provision 'configure-lustre-network', type: 'shell', run: 'never', inline: <<-SHELL
+    systemctl stop firewalld; systemctl disable firewalld;
+    systemctl start ntpd;
+    modprobe lnet
+    lnetctl lnet configure
+    lnetctl net add --net tcp0 --if enp0s9
```
Comment (jgrund) [inline iml-sandbox/Vagrantfile]: Are you sure it's always this interface?

Comment (johnsonw) [reply iml-sandbox/Vagrantfile]: Will update to eth1

File: iml-sandbox/Vagrantfile [hunk 3]
```diff
@@ -110,8 +110,7 @@ def configure_lustre_network(config)
     systemctl start ntpd;
     modprobe lnet
     lnetctl lnet configure
-    lnetctl net add --net tcp0 --if enp0s9
-    /sbin/modprobe zfs;
+    lnetctl net add --net tcp0 --if eth1
     genhostid
```
Comment (jgrund) [inline iml-sandbox/Vagrantfile]: This isn't needed here.

File: iml-sandbox/Vagrantfile [hunk 4]
```diff
@@ -110,8 +110,7 @@ def configure_lustre_network(config)
     systemctl start ntpd;
```
Comment (jgrund) [inline iml-sandbox/Vagrantfile]: Don't think we need the `firewalld` commands above.

File: iml-sandbox/Vagrantfile [hunk 5]
```diff
@@ -104,6 +104,15 @@ def fix_hostfile(config)
                       inline: "sed  -i '/^127.0.0.1\t#{config.vm.hostname}*/d' /etc/hosts"
 end
 
+def configure_lustre_network(config)
+  config.vm.provision 'configure-lustre-network', type: 'shell', run: 'never', inline: <<-SHELL
+    systemctl start ntpd;
```
Comment (jgrund) [inline iml-sandbox/Vagrantfile]: If we are going to start, should we also enable?

Comment (johnsonw) [reply iml-sandbox/Vagrantfile]: It couldn't hurt. I'll add it in.

**Pull request merged**